pom.xml

```diff
@@ -6,14 +6,14 @@
 
     <groupId>net.dkebnh.bukkit</groupId>
     <artifactId>MaintenanceMode</artifactId>
-    <version>0.3.0.b${env.BUILD_NUMBER}</version>
+    <version>0.3.0</version>
```

Comment:
I'd probably put the version back in so we can see the build number, maybe remove the `b` prefix though so it's just `0.3.0.${env.BUILD_NUMBER}`

Reply:
`env.BUILD_NUMBER` results in `null`, hence i removed it.
Im a bit confused as to how this gets a build number and what to use? Since we use Jenkins, isnt the build number assigned by Jenkins?
Looking through our other respositories, majority have either:

- `<version>0.3.0</version>` or 
- `<version>0.3.0-SNAPSHOT</version>`

Reply:
Yeah this will be null when building it yourself but when built by Jenkins, it will replace it with the current build number.

The version will be used in the plugin.yml and internally inside the plugin where necessary.

Reply:
updated to re-include build-number